examples/use-entire-request-body/README.md

```diff
@@ -0,0 +1,69 @@
+## Example: Use Entire Request Body
+
+This example demonstrates how a user can use `$(body)` in a [TriggerBinding](https://github.com/tektoncd/triggers/blob/master/docs/triggerbindings.md) to pass the entire request body as a single parameter. The example creates a TaskRun that prints the entire body of the HTTP request.
+
+### Try it out locally
+
+1. Create the ServiceAccount, Role, and RoleBinding:
+
+   ```shell script
+   kubectl apply -f examples/role-resources/triggerbinding-roles
+   kubectl apply -f examples/role-resources/
```

Comment:
I see a SA has been created here but i don't see the usage of the created SA in TaskRun